test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -37,7 +37,7 @@ public class VehicleController : Controller
         [HttpPut("/api/vehicles/{id}")]
         [Produces("application/json")]
         public object UpdateVehicleApi(
-            [Range(1, 500)] int id,
```

Comment:
`[Range]` is not our attribute and, like `[Validation]` and many a subclass, is completely able to target parameters.  aren't you introducing a regression from MVC 5?